### PR TITLE
Correction for dateTimeImmutable happens with #547

### PR DESCRIPTION
**Pull request by fferriere**
When we execute this test :
```
$this->dateTime($value = new \DateTimeImmutable())
                ->isEqualTo($value);
```

we have this error :
```
Instance of \dateTime is undefined
```

### CODE REVIEW

File: tests/units/classes/asserters/dateTime.php [hunk 1]
```diff
@@ -472,5 +472,12 @@ public function testIsImmutableLt55(atoum\locale $locale)
 						->hasMessage($notImmutable)
 			;
 		}
+		
+		/** @php >= 5.5.0 */
+		public function testClassicUsageImmutableDateTime()
+		{
+            $this->dateTime($value = new \DateTimeImmutable())
```
Comment (Hywan) [inline tests/units/classes/asserters/dateTime.php]: Please, fix CS here :-).

Comment (fferriere) [inline tests/units/classes/asserters/dateTime.php]: Hi,
I have correct it.
Sorry

File: tests/units/classes/asserters/dateTime.php [hunk 2]
```diff
@@ -472,5 +472,12 @@ public function testIsImmutableLt55(atoum\locale $locale)
 						->hasMessage($notImmutable)
 			;
 		}
+
+		/** @php >= 5.5.0 */
+		public function testClassicUsageImmutableDateTime()
+		{
+			$this->dateTime($value = new \DateTimeImmutable())
```
Comment (Hywan) [inline tests/units/classes/asserters/dateTime.php]: Also, in atoum' spirit, we will more likely write:
```php
$this
    ->dateTime($value = …)
        ->isEqualTo($value);
```

Sorry about that…

**Comment by Hywan** [comment]
Looks good for me after the last typo. Waiting for another peer review (like @jubianchi, @mageekguy or @mikaelrandy). Also, as a reminder for me: Do not forget to update the `CHANGELOG.md` file.

File: tests/units/classes/asserters/dateTime.php (hunk 2 above)
Comment (fferriere) [inline tests/units/classes/asserters/dateTime.php]: I have correct typo.
Sorry

**Comment by Hywan** [comment]
Thank you for your contribution @fferriere :-).

**Comment by fferriere** [comment]
You're welcome.
Thank atoum's team for your great work.

**Comment by mikaelrandy** [comment]
:+1:

**Comment by Hywan** [comment]
@fferriere Sorry, your branch is not out-dated. Can you rebase it please? Are you familiar with Git?

**Comment by fferriere** [comment]
Hi @Hywan,
I used the "Update branch" on github. It's OK ?

**Comment by Hywan** [comment]
@fferriere I don't know. I don't know this button but it looks fine :-).

**Comment by Hywan** [comment]
Thanks!

**Pull request merged**